### PR TITLE
Tweaks to enable using as test jar in syndesis tests

### PR DESCRIPTION
**Pull request by asmigala**

### CODE REVIEW

File: src/test/java/apicurito/tests/configuration/TestConfiguration.java [hunk 1]
```diff
@@ -95,6 +97,10 @@ public static String doReinstall() {
         return get().readValue(OPENSHIFT_REINSTALL, "true");
     }
 
+    public static String getAppRoot() {
+        return get().readValue(APP_ROOT, "app-root");
```
Comment (asmigala) [inline src/test/java/apicurito/tests/configuration/TestConfiguration.java]: Not sure about setting the default here

File: src/test/java/apicurito/tests/steps/MainPageSteps.java [hunk 2]
```diff
@@ -81,7 +81,7 @@ public void changeDescriptionTo(String description) {
     @And("^add contact info$")
     public void addContactInfo(DataTable table) {
         CommonUtils.getClickableLink(CommonUtils.Sections.CONTACT, MainPageUtils.CONTACT_SECTION).click();
-        for (List<String> dataRow : table.raw()) {
+        for (List<String> dataRow : table.cells()) {
```
Comment (asmigala) [inline src/test/java/apicurito/tests/steps/MainPageSteps.java]: @mmajerni haven't checked that this won't break your tests

**Comment by asmigala** [comment]
@mmajerni @mcada  please review (don't have rights to add you as reviewers)

**Comment by asmigala** [comment]
Will need some more tweaks

**Comment by asmigala** [comment]
@mmajerni could you please try to run your tests with these changes? If they pass, could you please merge?

**Comment by tplevko** [comment]
@mmajerni have you tried the tests with the changes?

**Comment by tplevko** [comment]
@asmigala there are conflicts that need to be resolved

**Comment by asmigala** [comment]
@tplevko closing this PR, the changes here were only for testing 7.3 syndesis and are already incorporated in @mmajerni 's PR

**Pull request closed without merging**